testing/tests/search.test.js

```diff
@@ -45,7 +45,7 @@ describe("Site search", () => {
     await expect(page).not.toMatchElement(".nothing-found", {
       text: "nothing found",
     });
-    await expect(page).toFill(SEARCH_SELECTOR, "/wboo");
+    await expect(page).toFill(SEARCH_SELECTOR, "/wbfoo");
```

Comment:
This scares me. Why can't `/en-US/docs/Web/Foo` be found when typing `/` followed by `wboo`?
Seems we might potentially have a bug then.

Reply:
Looks like it changes because the number of leading characters from the path that fuzzyjs can find in the query dropped from two ("/", "W") to one ("W"). Interestingly "wbfo" also works because it still matches two leading characters in the path ("W", "F"). 

I'll take a look at the other PR you opened sometime later today but I'm not sure this change actually makes real-world searches worse.

Reply:
Search can sometimes be "loosey goosey" and less of an exact science. 
But also, this is where your help is so useful because you can test it out on a bunch of example searches. Perhaps the best thing is to land *something* and see how it feels over time. 

By the way, just so you know, this fancy auto-complete search (which the URL-based fuzzy search is part of) is going to be disabled once we launch Yari. It's to make the going-live transition easier.